### PR TITLE
Feature/tables consistency

### LINKED ISSUE #1796: Reuse table component on every table view
Currently, table views could be more consistent in looks and functionality. 

- [x] incidents
- [ ] reports 
- [x] entities
- [x] users
- [ ] classifications
- [ ] variants

### LINKED ISSUE #1897: Table View – Column issues
Columns of text fields are too narrow by default, and width is not adjustable

image

### PR DESCRIPTION
**Pull request by clari182**
- closes #1897 
- closes #1796 

https://deploy-preview-1907--staging-aiid.netlify.app/apps/variants
https://deploy-preview-1907--staging-aiid.netlify.app/apps/reports
https://deploy-preview-1907--staging-aiid.netlify.app/apps/classifications

### CODE REVIEW

File: site/gatsby-site/src/components/variants/VariantsTable.js [hunk 1]
```diff
@@ -156,28 +132,31 @@ export default function VariantsTable({ data, refetch, setLoading }) {
   };
 
   const columns = React.useMemo(() => {
-    const columns = [
+    let columns = [
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/variants/VariantsTable.js:135]: Doesn't seem you need this change.

File: site/gatsby-site/src/components/ui/Table.js [hunk 2]
```diff
@@ -67,7 +69,142 @@ export function DefaultDateCell({ cell }) {
...
+
+  if (dateObj instanceof Date && !isNaN(dateObj)) {
+    return <>{format(new Date(dateObj), 'yyyy-MM-dd')}</>;
+  } else {
+    return <>{date}</>;
+  }
+}
+
 export default function Table({ table, className = '', ...props }) {
+  console.log(className);
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/ui/Table.js:207]: Please remove this

**Review by cesarvarela — COMMENTED** [review]
@clari182 please whitelist the deploy preview

**Comment by clari182** [comment]
@cesarvarela ready for another review

**Pull request merged**